docs/source/configuration/telemetry/instrumentation/standard-instruments.mdx

```diff
@@ -57,10 +57,14 @@ The coprocessor operations metric has the following attributes:
 ### Performance
 
 - `apollo_router_processing_time` - Time spent processing a request (outside of waiting for external or subgraph requests) in seconds.
-- `apollo_router_query_planning_time` - Time spent planning queries in seconds.
-- `apollo_router_query_planning_warmup_duration` - Time spent planning queries in seconds.
 - `apollo_router_schema_load_duration` - Time spent loading the schema in seconds.
 
+### Query planning
+- `apollo_router.query_planning.warmup.duration` - Time spent warming up the query planner queries in seconds.
+- `apollo.router.query_planner.plan.duration` - Histogram of plan durations.
+- `apollo.router.query_planner.total.duration` - Histogram of plan durations including queue time.
+- `apollo.router.query_planner.queued` - A gauge of the number of queued plans requests.
```

Comment:
`query_planning` or `query_planner` ?

Reply:
@garypen It's suggested to me that it _should_ have been `query_planner` and that `query_planning` was a regretted decision.

Reply:
Adjusting to `query_planning` as we now have both `query_planning` and `query_planner` keys in config.

Reply:
In future we should try and keep verbs out of top level metric names as we typically want to include the verb on a sublevel.